packages/sol-cov/src/coverage_subprovider.ts

```diff
@@ -115,29 +118,52 @@ export class CoverageSubprovider extends Subprovider {
     private async _recordTxTraceAsync(address: string, data: string | undefined, txHash: string): Promise<void> {
         let payload = {
             method: 'debug_traceTransaction',
-            params: [txHash, { disableMemory: true, disableStack: true, disableStorage: true }], // TODO For now testrpc just ignores those parameters https://github.com/trufflesuite/ganache-cli/issues/489
+            params: [txHash, { disableMemory: true, disableStack: false, disableStorage: true }],
         };
         const jsonRPCResponsePayload = await this.emitPayloadAsync(payload);
         const trace: TransactionTrace = jsonRPCResponsePayload.result;
-        const coveredPcs = _.map(trace.structLogs, log => log.pc);
         if (address === constants.NEW_CONTRACT) {
-            const traceInfo: TraceInfoNewContract = {
-                coveredPcs,
-                txHash,
-                address: address as 'NEW_CONTRACT',
-                bytecode: data as string,
-            };
-            this._coverageManager.appendTraceInfo(traceInfo);
+            const tracesByContractAddress = getTracesByContractAddress(trace.structLogs, address);
+            for (const subcallAddress of _.keys(tracesByContractAddress)) {
+                let traceInfo: TraceInfoNewContract | TraceInfoExistingContract;
+                if (subcallAddress === 'NEW_CONTRACT') {
+                    const traceForThatSubcall = tracesByContractAddress[subcallAddress];
+                    const coveredPcs = _.map(traceForThatSubcall, log => log.pc);
+                    traceInfo = {
+                        coveredPcs,
+                        txHash,
+                        address: address as 'NEW_CONTRACT',
+                        bytecode: data as string,
+                    };
+                } else {
+                    payload = { method: 'eth_getCode', params: [subcallAddress, BlockParamLiteral.Latest] };
+                    const runtimeBytecode = (await this.emitPayloadAsync(payload)).result;
```

Comment:
Split into two lines please.

Also, any chance this doesn't return the `runtimeBytecode`? If so, what is returned?

Reply:
There is no such chance